src/python/dxpy/scripts/dx.py

```diff
@@ -4881,6 +4884,7 @@ def register_parser(parser, subparsers_action=None, categories=('other', ), add_
                         help=fill('Print help and examples for how to specify inputs',
                                   width_adjustment=-24),
                         action=runInputHelp, nargs=0)
+parser_run.add_argument('--detach', help=fill("New job will start as a detached job if run inside of a different job.", width_adjustment=-24), action='store_true')
```

Comment:
So maybe: "as a detached job (not a subjob) if run ..".

Reply:
```suggestion
parser_run.add_argument('--detach', help=fill("When invoked from a job, detaches the new job from the creator job so the new job will appear as a typical root execution.  Setting DX_RUN_DETACH environment variable to 1 causes this option to be set by default.", width_adjustment=-24), action='store_true')
```